librz/core/cbin.c

```diff
@@ -4673,7 +4679,7 @@ RZ_API char *rz_core_bin_method_flags_str(ut64 flags, int mode) {
 	return rz_strbuf_drain(buf);
 }
 
-RZ_API RzCmdStatus rz_core_bin_plugin_print(const RzBinPlugin *bp, RzCmdStateOutput *state) {
+RZ_IPI RzCmdStatus rz_core_bin_plugin_print(const RzBinPlugin *bp, RzCmdStateOutput *state) {
```

Comment:
since these are IPI i would remove the `rz_` prefix
```suggestion
RZ_IPI RzCmdStatus core_bin_plugin_print(const RzBinPlugin *bp, RzCmdStateOutput *state) {
```

Reply:
Almost all of our `RZ_IPI` start with `rz_`: https://github.com/rizinorg/rizin/blob/dev/librz/core/core_private.h
Also be aware that this change is not directly related to the pr here. It's only to fix the windows build after including `core_private.h` above because there was a mismatch between that header and the definition.